common/web/input-processor/build.sh

```diff
@@ -70,7 +70,7 @@ if builder_start_action build:tools; then
 fi
 
 if builder_start_action build:module; then
-  npm run tsc -- -b src/tsconfig.json
+  tsc -b src/tsconfig.json
```

Comment:
Could we please document the first instance of changes like this with a reference to `set_keyman_standard_build_path`?  This kind of buries the lede a bit on how path resolution works; external folks used to `npm -g <package>` installations might be surprised that this call will be repo-local despite visually using global-installation syntax.

Reply:
Is your comment here sufficient documentation? :grin:

Reply:
```
## Standard environment

`build-utils.sh` will prepend `$KEYMAN_ROOT/node_modules/.bin` to the `PATH`
variable to ensure that we run the correct versions of npm package commands, so
there is no need to hard-code path references or add script wrappers to
package.json (`npm run <script>`).

Other environment variables and paths will probably be added over time.
```